profiler/__init__.py

```diff
@@ -0,0 +1,94 @@
+# coding: utf-8
+from . import controllers  # noqa
```

Comment:
As for the import itself, I don't remember (it's been so long) if importing controllers is really necessary. 

That being said, if the question is about the noqa tag, that was because we were using a linter (flake8, i.e pyflakes) that would normally complain about "unused" imports, and we don't want to disable that check globally.

In the OCA context, the situation might be different, yours to judge !